pkg/internalapis/register.go

```diff
@@ -86,6 +87,12 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&cnscsisvfeaturestatesv1alpha1.CnsCsiSvFeatureStatesList{},
 	)
 
+	scheme.AddKnownTypes(
+		SchemeGroupVersion,
```

Comment:
we are using the same API version for all APIs? In the future when we need to upgrade the API version of one CR, type we will need to do refactoring. 

Can we keep the version specific to API and not use the same version variable for all APIs? Refer https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/pkg/apis/migration API

Reply:
Done